app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
> if we are still using that exact file 

We aren't using "exact file" for any files. What I'm wondering about is how we should attribute code that has been referenced.

Let me give you an example:

Nov has a file called `openid_connect_sample/lib/authentication.rb`. Essentially everything is besides the following method is not used/referenced (mostly because the way Nov handles users in his applications is different from ours): https://github.com/nov/openid_connect_sample/blob/master/lib/authentication.rb#L56 

In our application it is here (slightly modified to fit the relationship between our authorization and token model): https://github.com/diaspora/diaspora/pull/6095/files#diff-fedfb5bf76edad6062d6c81ade1babd5R6